src/yashCJun20/Assignment20/ReverseNumber.java

```diff
@@ -0,0 +1,21 @@
+package yashCJun20.Assignment20;
+
+public class ReverseNumber {
+	
+
+	public static void main(String[] args) {
+		
+		int number=12345;
+		int revNumber=0;
+		System.out.println("Reverse Number is");
+		while (number!=0) {
+			
+			revNumber=number % 10;
+			number=number/10;
+			System.out.print(revNumber);
```

Comment:
here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your login.